### PR TITLE
Annotate markdown fields in API and database

### PR DESCRIPTION
**Pull request by kalvdans**
Patch that sprinkles "CommonMark without images" everywhere, feel free to thumbsdown if you think it pollutes the files rather than helping.

Not sure if references are to allowed to be Markdown formatted or just a string of words.

### CODE REVIEW

**Comment by lucaslcode** [comment]
Maybe this should wait until after #529 to avoid merge conflicts

File: app/backend/src/couchers/models.py
Comment (aapeliv) [inline app/backend/src/couchers/models.py]: I believe the group chat messages are plaintext. Is that right @lucaslcode?

File: app/backend/src/couchers/models.py
Comment (aapeliv) [inline app/backend/src/couchers/models.py]: The reference also is currently plaintext, I think

File: app/pb/api.proto
Comment (aapeliv) [inline app/pb/api.proto]: As above

File: app/pb/api.proto
Comment (aapeliv) [inline app/pb/api.proto]: As above

File: app/pb/conversations.proto
Comment (aapeliv) [inline app/pb/conversations.proto]: As above

**Review by aapeliv — CHANGES_REQUESTED** [review]

**Comment by aapeliv** [comment]
> Maybe this should wait until after #529 to avoid merge conflicts

#529 has now been merged.

**Comment by kalvdans** [comment]
> #529 has now been merged.

Thanks for the reminder! Merged latest develop and fixed your comments.

File: app/pb/conversations.proto
Comment (aapeliv) [reply app/pb/conversations.proto]: I think this is still plain text

**Review by aapeliv — CHANGES_REQUESTED** [review]

**Review by aapeliv — APPROVED** [review]
LGTM!

**Pull request merged**
